Makefile

```diff
@@ -1,15 +1,15 @@
 .PHONY: build test-build test-in-container push exec-test clean
 
-IMAGE_NAME = docker.io/usercont/packit-generator
-TEST_IMAGE_NAME = docker.io/usercont/packit-generator-test
+IMAGE_NAME = docker.io/usercont/sandcastle
+TEST_IMAGE_NAME = docker.io/usercont/sandcastle
```

Comment:
The same name as for `IMAGE_NAME` as for `TEST_IMAGE_NAME`. I guess typo.

Reply:
oh right, good catch!